templates/express_pg_sequelize/index.js

```diff
@@ -0,0 +1,38 @@
+import express from 'express';
+import db from './db.js';
+import dotenv from 'dotenv'
+
+dotenv.config()
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.json());
+
+// Test PostgreSQL connection
+db.authenticate()
+  .then(() => {
+    console.log('Database connection established successfully.');
+  })
+  .catch((err) => {
+    console.error('Unable to connect to the database:', err);
+  });
+
+app.get('/', (req, res) => {
+  res.send('Hello, Express with PostgreSQL!');
+});
+
+// Dummy route to test DB query
+app.get('/db-test', async (req, res) => {
+  try {
+    const [results, metadata] = await db.query('SELECT NOW()');
+    res.status(200).json({ message: 'DB query successful!', data: results });

```

Comment:
Using a `return` will prevent us from this error `Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client`.
Another supporting article: https://dev.to/adamkatora/why-you-should-always-use-return-before-ressend-in-express-apis-and-applications-k9k

```suggestion
    return res.status(200).json({ message: 'DB query successful!', data: results });
```